ocs_ci/ocs/bucket_utils.py

```diff
@@ -265,6 +265,23 @@ def copy_random_individual_objects(
         logger.info(f"Copied {src_obj}")
 
 
+def upload_objects_with_javasdk(javas3_pod, s3_obj, bucket_name, is_multipart=False):
+    """
+    Runs upload operation using java s3 pod
+    """
+    access_key = s3_obj.access_key_id
+    secret_key = s3_obj.access_key
+    endpoint = s3_obj.s3_internal_endpoint
+    command = (
+        'mvn exec:java -Dexec.mainClass=amazons3.s3test.ChunkedUploadApplication -Dexec.args="'
+        + f"{endpoint} {access_key} {secret_key} {bucket_name} {is_multipart}"
+        + '" -Dmaven.test.skip=true package'
+    )
+    javas3_pod.exec_cmd_on_pod(
+        command=command, out_yaml_format=False
+    ), "Failed to upload objects!"
```

Comment:
Missing assert?

Reply:
Nope thats taken care inside the test function